training/movinet_with_hmdb51/Classify-Vanilla-trained.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c9b8f6e9-668c-4c4b-81dd-5fcfb7956d51",
+   "metadata": {},
+   "source": [
+    "# Classification with a vanilla model vs trained model.\n",
+    "\n",
+    "## Install pre requisites"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "064eb8e6-054d-4f9a-bf97-903706a577b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q git+https://github.com/Atze00/MoViNet-pytorch.git\n",
+    "!pip install -q av\n",
+    "!pip install -q -U aperturedb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5382ab3d-dea0-4507-bf41-bcea892a6377",
+   "metadata": {},
+   "source": [
+    "## Util functions\n",
+    "\n",
+    "### Load datasets as clips (of 16 frames), sampled at 5fps"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4871f56-82ef-4aaa-99cd-91cc9f36db27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torchvision.transforms import v2 as T\n",
+    "import torch\n",
+    "\n",
+    "def get_common():\n",
+    "    \"\"\"\n",
+    "    Just common parameters.\n",
+    "    Applies to the training and data loading sections.\n",
```

Comment:
Code does need more comments but its looking good. Are you planning to land the READMe in docs?